developer-docs-site/docs/sdks/rust-sdk.md

```diff
@@ -5,11 +5,16 @@ slug: "rust-sdk"
 
 # Aptos Rust SDK
 
-Aptos provides an official Rust SDK. The Rust SDK is tested carefully, though it isn't as popular as the [Typescript SDK](/sdks/typescript-sdk).
+### Installing Rust SDK
```

Comment:
`##` instead, since the next heading up is `#`?

Reply:
For the time I am breaking our own styleguide rule. Other index files have `###` so for the timebeing I will keep it as 3. Later on I will fix all occurrences of such.